### ISSUE TITLE
Texture behavior under saturated light

### ISSUE THREAD

**Issue by fabienrohrer** [x3dom/x3dom#295]
Hi,

I'm working on an X3D exporter, and I'm blocking on a problem about saturated enlightenment and the corresponding textured objects behavior. This difference let me stuck in my project to use X3dom.

When multiplying the lights from this world:

```
<x3d id="x3d" width='640px' height='480px' showLog='false'>
<Scene>
<NavigationInfo headlight="false"></NavigationInfo>
<Viewpoint orientation='0.727587 0.65891 0.190934 5.3185' position='-1.43383 2.00337 1.94946'></Viewpoint>
<Background skyColor='0.4 0.7 1'></Background>
<PointLight location='0.5 0.5 0'></PointLight>
<Transform><Shape><Appearance><Material diffuseColor='1 1 1'></Material><ImageTexture url='"textures/parquetry.png"' origChannelCount='3'></ImageTexture></Appearance><IndexedFaceSet coordIndex='0 1 2 3 -1' texCoordIndex='0 1 2 3 -1'><Coordinate point='-0.5 0 -0.5, -0.5 0 0.5, 0.5 0 0.5, 0.5 0 -0.5'></Coordinate><TextureCoordinate point='0 1, 0 0, 1 0, 1 1'></TextureCoordinate></IndexedFaceSet></Shape></Transform>
<Transform translation='1 0 0'><Shape><Appearance><Material diffuseColor='1 0 0.0117952'></Material></Appearance><IndexedFaceSet coordIndex='0 1 2 3 -1' texCoordIndex='0 1 2 3 -1'><Coordinate point='-0.5 0 -0.5, -0.5 0 0.5, 0.5 0 0.5, 0.5 0 -0.5'></Coordinate><TextureCoordinate point='0 1, 0 0, 1 0, 1 1'></TextureCoordinate></IndexedFaceSet></Shape></Transform>
</Scene>
</x3d>
```

I get the following behaviors:
- my software - 2 lights
  ![saturated_webots](https://f.cloud.github.com/assets/866788/2097165/cd95bd0c-8f13-11e3-92e9-3226130eadee.png)
- x3dom - 2 lights
  ![saturated_x3dom](https://f.cloud.github.com/assets/866788/2097167/d2cc0240-8f13-11e3-9e56-b628d8602edb.png)
- my software - 10 lights
  ![highly_saturated_webots](https://f.cloud.github.com/assets/866788/2097169/d8509a6e-8f13-11e3-9da5-41ca8535ef02.png)
- x3dom - 10 lights
  ![highly_saturated_x3dom](https://f.cloud.github.com/assets/866788/2097173/e3a97070-8f13-11e3-9f94-3d6b5a358abd.png)

To summarize, when the light is saturated, textured objects approach white for x3dom, and diffuseColor for my software.

In my point of view, a regular texture is a diffuse color map, and should behave as a regular diffuseColor. This leads me to think my implementation is correct. But I have troubles to find that in the X3D specifications.

My questions:
1. Who implements correctly the X3D specifications there?
2. Which OpenGL parameters are responsible of this?

Sincerely
Fabien
